numba/cuda/tests/cudapy/test_compiler.py

```diff
@@ -192,15 +193,37 @@ def test_c_abi_with_abi_name(self):
         self.assertRegex(ptx, r"\.visible\s+\.func\s+\(\.param\s+\.b32\s+"
                               r"func_retval0\)\s+_Z4funcii\(")
 
+    def test_compile_defaults_to_c_abi(self):
+        ptx, resty = compile(f_module, int32(int32, int32), device=True)
+
+        # The function name should match the Python function name, and its
+        # return value should be 32 bits
+        self.assertRegex(ptx, r"\.visible\s+\.func\s+\(\.param\s+\.b32\s+"
+                              r"func_retval0\)\s+f_module\(")
+
+    def test_compile_to_ltoir(self):
+        if runtime.get_version() < (11, 5):
+            self.skipTest("-gen-lto unavailable in this toolkit version")
+
+        ltoir, resty = compile(f_module, int32(int32, int32), device=True,
```

Comment:
`resty` is unused?

Reply:
It is, but I don't like anonymizing unused returned tuple values by assigning them to `_` or similar, because then it's not obvious what's being discarded if you don't know the API (`compile()`, in this case) very well.

Reply:
I see, could you assert is has an expected value perhaps?